metrics/doc.go

```diff
@@ -0,0 +1,38 @@
+/*
+Package metrics implements collection of common performance metrics.
+
+It uses the Go implementation of the Coda Hale metrics library:
+
+https://github.com/dropwizard/metrics
+https://github.com/rcrowley/go-metrics
+
+The collected metrics include detailed information about Skipper's relevant processes while serving requests -
+looking up routes, filters (aggregate and individual), backend communication and forwarding the response to
+the client.
+
+Options
+
+To enable metrics, it needs to be initialized with a Listener address.In this case, Skipper will start an additional
+http listener, where the current metrics values can be downloaded.
+
+You can define a custom Prefix to every reported metrics key. This allows you to avoid conflicts between Skipper's
+metrics and other systems if you aggregate them later in some monitoring system. The default prefix is "skipper."
+
+You can also enable some Go garbage collector and runtime metrics using EnableDebugGcMetrics and EnableRuntimeMetrics,
+respectively.
+
+REST API
+
+This listener accepts GET requests on the /metrics endpoint like any other REST api. A request to "/metrics" should
```

Comment:
Yes, this is related to the next statement. If you don't register debug gc or runtime stats and the instance didn't serve a single request it will return a 404 because it still has no metrics.

Maybe I should make this part more clear - no known metrics (yet) resulting in a 404.

What would you suggest?